ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -1474,8 +1474,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
-- as: gcp-ipi-oidc-arm-f28-destructive
-  cron: 24 10 6 * *
+- as: gcp-ipi-oidc-mini-perm-arm-f28-destructive
```

Comment:
Sounds like we missed non-destructive job before, do you mind to add it together for now.